scripts/check_organizations.py

```diff
@@ -25,12 +25,13 @@ def main(directory: Path) -> int:
         report = get_organization_validation_report(org_path)
 
         if not report.valid:
+            print(f"Organization described in {org_path} is not valid \n")
```

Comment:
(Je ne sais pas si c'est volontaire mais par défaut `print()` ajoute déjà un retour à la ligne)

```suggestion
            print(f"Organization described in {org_path} is not valid")
```